ui/docs/reference/core/web-server/README.mdx

```diff
@@ -23,8 +23,8 @@ Again, this approach is intended only for use by the Taskcluster UI.
 
 The following login strategies are supported:
 
-* [GitHub](github-login-strategy.md)
-* [Mozilla-Auth0](mozilla-auth0-login-strategy.md)
+* [GitHub](github-login-strategy.mdx)
+* [Mozilla-Auth0](mozilla-auth0-login-strategy.mdx)
```

Comment:
Interesting -- in fact these should not have extensions at all (just the filename)

Reply:
I think these are pointing to these files
https://github.com/taskcluster/taskcluster/blob/master/ui/docs/reference/core/web-server/github-login-strategy.md
https://github.com/taskcluster/taskcluster/blob/master/ui/docs/reference/core/web-server/mozilla-auth0-login-strategy.md

These files are renamed to `mdx` in this PR

Reply:
What happens if you remove the `.mdx` extension. Does the page still render?

Reply:
It shows 404 page on clicking on any of the option if we change names to `md`

Reply:
I don't seem to be able to view the links on the preview site. Can you replicate?

https://deploy-preview-1331--taskcluster-web.netlify.com/docs/reference/core/web-server

Reply:
The existing `.md` links don't work, either -- the links are already broken before this PR.  All of the links between pages on the docs site omit the extension, and whoever wrote this (probably me) forgot that and didn't check the links before making a PR (yeah, sounds just like me).

Reply:
https://deploy-preview-1331--taskcluster-web.netlify.com/docs/reference/core/web-server/github-login-strategy

https://deploy-preview-1331--taskcluster-web.netlify.com/docs/reference/core/web-server/mozilla-auth0-login-strategy

These are the links on preview site and it seems working

I have updated the PR with removed mdx extension

Reply:
https://deploy-preview-1331--taskcluster-web.netlify.com/docs/reference/core/web-server/github-login-strategy

https://deploy-preview-1331--taskcluster-web.netlify.com/docs/reference/core/web-server/mozilla-auth0-login-strategy

These are the links on preview site and it seems working

I have updated the PR with removed mdx extension